### PR TITLE
add descriptions for background pictures

### LINKED ISSUE #40689: [BITV]: Replace `aria-label`'s of background buttons with another name
This is accessibility issue.

For now background buttons have `aria-label`'s which are not really understandable, like "Soft floral (Hannah MacLean, CC0)". The `aria-label` have to be like "Background with abstract ..."

Each name have to unique.

Screenshot from 2023-09-28 18-51-42

### PR DESCRIPTION
**Pull request by szaimen**
Close https://github.com/nextcloud/server/issues/40689

### CODE REVIEW

File: apps/theming/lib/Service/BackgroundService.php [hunk 1]
```diff
@@ -57,101 +57,120 @@ class BackgroundService {
 	public const SHIPPED_BACKGROUNDS = [
 		'hannah-maclean-soft-floral.jpg' => [
 			'attribution' => 'Soft floral (Hannah MacLean, CC0)',
+			'description' => 'Abstract picture in yellow and white color whith a flower on it',
```
Comment (szaimen) [inline apps/theming/lib/Service/BackgroundService.php:60]: Please review these descriptions carefully. Are they good or should they rather be something like `Abstract background picture ...` ? (So should I add the word `background` everywhere?

Comment (JuliaKirschenheuter) [reply apps/theming/lib/Service/BackgroundService.php:60]: I would vote for `Abstract background picture ...`

Comment (szaimen) [reply apps/theming/lib/Service/BackgroundService.php:60]: okay adding the word background then everywhere

**Review by JuliaKirschenheuter — APPROVED** [review]
Looks good, thank you!

File: apps/theming/lib/Service/BackgroundService.php (hunk 1 above)
Comment (szaimen) [reply apps/theming/lib/Service/BackgroundService.php]: done

Comment (susnux) [reply apps/theming/lib/Service/BackgroundService.php]: Well I would have said the opposite as background image is no description of what you see on it.
Moreover you should avoid terms like "picture of" or "image of" as it is already clear that it is an image. 

But I am pretty bad at describing images so fine for me - better than before:)

**Review by susnux — APPROVED** [review]

**Pull request merged**